spec/acceptance/api/message_spec.rb

```diff
@@ -9,9 +9,14 @@
   api 'message' do
     post '/admin/api/accounts/:account_id/messages.:format', action: :create do
       parameter :body, 'Message Body'
+      parameter :subject, 'Message Subject'
       let(:body) { 'This is some text' }
+      let(:subject) { 'custom subject' }
 
-      after { serializable.body.should == body }
+      after do
+        serializable.body.should == body
+        serializable.subject.should == subject
```

Comment:
I'm confused here again. Is `subject` here the parameter subject or the `let(:subject)`? And if it is `let(:subject)`, then why parameter `subject` is ignored?